.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  test:
+    env:
+      MIX_ENV: test
+    runs-on: ubuntu-latest
+    services:
+      db:
+        image: postgres:10
+        ports: ['5432:5432']
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
Are _actions_ based on docker images?

Reply:
This is not a reference to a Docker image but to a GitHub repo: https://github.com/actions/checkout